Lib/fontTools/otlLib/builder.py

```diff
@@ -428,8 +430,8 @@ def buildSinglePos(mapping, glyphMap):
     # is unique as well. We encode these in format 1 again.
     for key, glyphs in coverages.items():
         if key not in handled:
-            assert len(glyphs) == 1, glyphs
-            st = buildSinglePosSubtable({glyphs[0]: values[key]}, glyphMap)
+            for g in glyphs:
+                st = buildSinglePosSubtable({g: values[key]}, glyphMap)
```

Comment:
you removed the assertion that `len(glyphs) == 1` but the comment still says that remaining ValueRecords are supposed to be used by a single glyph..

Reply:
comment adjusted.